docs/cloud-services-config.md

```diff
@@ -154,21 +156,113 @@ Left navigation now supports extra nesting level. To add additional level to you
   
 ```
 
-### All services dropdown
+# All services dropdown and page<a name="allservices"></>
 
 There is a new schema that controls which items will be display in the all services dropdown and pages.
 
-The data is store in `service.json` files within `static` directory. Each environment has its own definition file. For example, for `stage-beta` environment, the file is located at `/static/beta/stage/services/services.json`
+The data is stored in `service.json` files within `static` directory. Each environment has its own definition file. For example, for `stage-beta` environment, the file is located at `/static/beta/stage/services/services.json`
+
+**All the tiles (links) in the all services dropdown (page), must exist somewhere in the navigation files**. The tiles (links) are generated from these files. If no reference is found in the navigation files, the tile (link) will not show in the UI.
+
+**Changes to the navigation files can affect the all services dropdown (page)**. Before you submit the changes for review, make sure the all services will be in expected state.
 
-#### Link requirements
+Be aware that changing the link IDs in navigation file and not reflecting the change in respective `services.json` will remove the tile from the all services dropdown (page).
+
+## Link requirements
 
 In order for a link to be eligible to display in the all services dropdown it must have following attributes inside the <bundle>-navigation.json file:
 
 - `id`: a bundle unique id of a **link**
 - `description`: short description of the page the link leads to
 - `href`: pathname to be used
 
-#### Inserting link into services.json
+### Using expandable section as a link
+
+If your leaf links (clickable links in left navigation) are not descriptive enough for the services dropdown or page, the expandable section can be used. **The HREF attribute will be picked from a first valid link of expandable section.**
```

Comment:
Add an example in here `(For instance systems in RHL is referenced multiple times, it's better to use parent ID)`